scripts/upstreamCommit.sh

```diff
@@ -1,38 +1,35 @@
-#!/usr/bin/env bash
-(
-set -e
-PS1="$"
+#!/usr/bin/env sh
+set -ue
+PS1="$" # TODO: Why?
```

Comment:
Performance optimization
Many people have complex PS1's that get processed by some commands, and this ensures it doesnt run your PS1